.rubocop.yml

```
@@ -0,0 +1,74 @@
+require: rubocop-rspec
+require: rubocop-thread_safety
+
+AllCops:
+  DisplayCopNames: true
+  Exclude:
+    - 'bin/**/*'
+    - 'config/**/*'
+    - 'db/**/*'
+    - 'node_modules/**/*'
+    - 'tmp/**/*'
+    - 'vendor/**/*'
+
+Bundler/OrderedGems:
+  Enabled: false
+
+Layout/AlignArray:
+  Enabled: false
+
+Layout/AlignParameters:
+  Enabled: false
+
+Layout/SpaceInsideArrayLiteralBrackets:
+  Enabled: false
+
+Metrics/AbcSize:
+  Enabled: false
+
+Metrics/BlockLength:
+  Exclude:
+    - "spec/**/*"
+
+Metrics/ClassLength:
+  Enabled: false
+
+Metrics/CyclomaticComplexity:
+  Enabled: false
+
+Metrics/LineLength:
+  Max: 120
+  Exclude:
+    - "spec/**/*"
+
+Metrics/MethodLength:
+  Max: 20
+
+Metrics/ModuleLength:
+  Enabled: false
+
+Metrics/ParameterLists:
+  CountKeywordArgs: false
+
+Performance:
+  Exclude:
+    - "spec/**/*"
+
+# ok to use and/or for flow control
+Style/AndOr:
+  EnforcedStyle: 'conditionals'
+
+Style/Documentation:
+  Enabled: false
+
+Style/ExpandPathArguments:
+  Enabled: false
+
+Style/MultilineBlockChain:
+  Enabled: false
+
+Style/StringLiterals:
+  Enabled: false
```

can we use double quotes everywhere?

```suggestion
  Enabled: true
  EnforcedStyle: double_quotes
```

this is consistent with [rails](https://github.com/rails/rails/blob/master/.rubocop.yml#L156-L159), [github](https://github.com/github/rubocop-github/blob/master/config/default.yml#L324-L326), [thoughtbot](https://github.com/thoughtbot/guides/blob/master/style/ruby/.rubocop.yml#L339-L343)

---

Seems reasonable!